webviz_subsurface/plugins/_tornado_plotter_fmu/shared_settings/_view_settings.py

```diff
@@ -0,0 +1,84 @@
+from typing import List
+
+import pandas as pd
+import webviz_core_components as wcc
+from dash.development.base_component import Component
+from webviz_config.utils import StrEnum
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC
+
+
+class FilterOption(StrEnum):
+    REMOVE_SENS_WITH_NO_IMPACT = "remove-sens-with-no-impact"
+
+
+class Scale(StrEnum):
+    REL_VALUE_PERC = "Relative value (%)"
+    REL_VALUE = "Relative value"
+    TRUE_VALUE = "True value"
+
+
+class ViewSettings(SettingsGroupABC):
+    class IDs(StrEnum):
+        REFERENCE = "reference"
+        SCALE = "scale"
+        SENSITIVITIES = "sensitivities"
+        FILTER_OPTIONS = "plot-options"
+
+    def __init__(
+        self,
+        realizations: pd.DataFrame,
+        reference: str = "rms_seed",
+        allow_click: bool = False,
+    ) -> None:
+        super().__init__("View Settings")
+
+        self.sensnames = list(realizations["SENSNAME"].unique())
+
+        self.scales = [Scale.REL_VALUE_PERC, Scale.REL_VALUE, Scale.TRUE_VALUE]
+
+        self.filter_options = [
+            {
+                "label": "Remove sensitivities with no impact",
+                "value": FilterOption.REMOVE_SENS_WITH_NO_IMPACT,
+            }
+        ]
+
+        self.initial_reference = (
+            reference if reference in self.sensnames else self.sensnames[0]
+        )
+
+        self.allow_click = allow_click
+
+    def layout(self) -> List[Component]:
+        return [
+            wcc.Dropdown(
+                id=self.register_component_unique_id(ViewSettings.IDs.REFERENCE),
+                label="Reference",
+                options=[{"label": r, "value": r} for r in self.sensnames],
+                value=self.initial_reference,
+                multi=False,
+                clearable=False,
+            ),
+            wcc.Dropdown(
+                id=self.register_component_unique_id(ViewSettings.IDs.SCALE),
+                label="Scale",
+                options=[{"label": r, "value": r} for r in self.scales],
+                value=self.scales[0],
```

Comment:
Use `Scale`-enum directly in for-loop? And set `value=Scale.REL_VALUE_PERC` instead?